engine/src/events/timed_event.cc

```diff
@@ -456,7 +444,6 @@ int timed_event::handle_timed_event() {
       &timed_event::_exec_event_expire_downtime,
       &timed_event::_exec_event_host_check,
       &timed_event::_exec_event_hfreshness_check,
-      &timed_event::_exec_event_reschedule_checks,
```

Comment:
Are you sure that follow event types had been redefined?

Reply:
This is an array of methods, I suspect they are get by their index. So if you remove this one, all the following ones are moved by 1.